src/open_clip/coca_model.py

```diff
@@ -220,7 +222,7 @@ def generate(
                         dim=1
                     )
                 return output
-            
+
             elif generation_type == "top_p":
                 logit_warper = GENERATION_TYPES[generation_type](top_p)
             elif generation_type == "top_k":
```

Comment:
Look like possible NPE `GENERATION_TYPES[generation_type]` -> `None` and `None(top_p)`

Reply:
Pushed separately